tests/network-tests/src/fixtures/membership/VerifyValidatorAccountFixture.ts

```diff
@@ -0,0 +1,78 @@
+import { Api } from '../../Api'
+import { assert } from 'chai'
+import { QueryNodeApi } from '../../QueryNodeApi'
+import { SubmittableExtrinsic } from '@polkadot/api/types'
+import { BaseQueryNodeFixture } from '../../Fixture'
+import { MemberContext, EventDetails } from '../../types'
+import { MembershipFieldsFragment, MemberProfileUpdatedEventFieldsFragment } from '../../graphql/generated/queries'
+import { MembershipMetadata } from '@joystream/metadata-protobuf'
+import { Utils } from '../../utils'
+
+export class VerifyValidatorAccountFixture extends BaseQueryNodeFixture {
```

Comment:
The fixture needs a `getExtrinsics` method, to allow it to actually send the transaction that it's meant to test, which is either `operationsWorkingGroupBeta.leadRemark(msg)` or `operationsWorkingGroupBeta.workerRemark(workerId, msg)`. You can check [this fixture](https://github.com/Joystream/joystream/blob/master/tests/network-tests/src/fixtures/forum/RemarkModeratePostsFixture.ts#L33) for example.